modules/message-lane/src/benchmarking.rs

```diff
@@ -70,23 +95,202 @@ benchmarks_instance! {
 		}
 		confirm_message_delivery::<T, I>(T::MaxMessagesToPruneAtOnce::get());
 
-		let (payload, fee) = T::prepare_message(MessageParams { size_factor: WORST_MESSAGE_SIZE_FACTOR });
+		let (payload, fee) = T::prepare_outbound_message(MessageParams { size_factor: WORST_MESSAGE_SIZE_FACTOR });
 	}: send_message(RawOrigin::Signed(sender), lane_id, payload, fee)
+	verify {
+		assert_eq!(
```

Comment:
I may add `test_benchmarks` to this module, but we are not testing benchmarks on CI anyway. So imo it doesn't makes sense. These verifications are also running during benchmark execution.